assets/app/styles/_core.less

```diff
@@ -301,6 +314,11 @@ html, body {
   }
 }
 
+.required:before {
+  content: '* ';
```

Comment:
is `.required` too general a class-name? do you want to scope this to `label.required`?